src/strategies/nation3-passport-coop-with-delegations/index.ts

```diff
@@ -29,28 +32,49 @@ export async function strategy(
 ): Promise<Record<string, number>> {
   const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
 
-  const erc721SignerCaller = new Multicaller(network, provider, signerAbi, {
-    blockTag
-  });
-  const erc721OwnerCaller = new Multicaller(network, provider, ownerAbi, {
-    blockTag
-  });
   const erc20BalanceCaller = new Multicaller(network, provider, balanceAbi, {
     blockTag
   });
+
   const erc721LastTokenIdCaller = new Multicaller(
     network,
     provider,
     lastTokenIdAbi,
     { blockTag }
   );
 
+  const erc721SignerCaller = new Multicaller(network, provider, signerAbi, {
+    blockTag
+  });
+  const erc721OwnerCaller = new Multicaller(network, provider, ownerAbi, {
+    blockTag
+  });
+
+  const passportIssuanceSubgrgraph = "https://api.thegraph.com/subgraphs/name/nation3/passportissuance";
```

Comment:
I think that this should be in configuration just because in theory this is a strategy that could be used by other projects, even if that is really unlikely! maybe just the `nation3/passsportissuance` part...